src/Axe.Windows.TelemetryTests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System.Reflection;
+using System.Runtime.CompilerServices;
```

Comment:
CompilerServices isn't needed here (it's only needed if you use InternalsVisibleTo)